package.json

```diff
@@ -36,10 +37,14 @@
     "babel-core": "^6.26.0",
     "babel-loader": "^7.1.2",
     "babel-plugin-transform-class-properties": "^6.24.1",
+    "babel-preset-es2015": "^6.24.1",
     "babel-preset-react": "^6.24.1",
     "express": "^4.16.2",
     "react": "^16.1.1",
     "react-dom": "^16.1.1",
-    "webpack": "^3.8.1"
+    "react-hot-loader": "^3.0.0",
+    "webpack": "^3.8.1",
+    "webpack-dev-server": "^2.9.4"
```

Comment:
Maybe move `react-hot-loader` and `webpack-dev-server` into `"devDependencies"` since they won't be needed on the Heroku server where the app is deployed.